app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ReviewsController < ApplicationController
+
+  before_action :require_user
+
+  def create
+    @video = Video.find(params[:video_id])
+    review = Review.new review_params.merge!(video_id: params[:video_id], user: current_user)
```

Comment:
And another way avoiding this concern could be just this `@video.reviews.build(review_params.merge!(user: current_user))`